core/jobs/src/test/java/com/cognifide/aet/job/common/comparators/layout/utils/ImageComparisonTest.java

```diff
@@ -95,6 +95,38 @@ public void testCompare_different() throws Exception {
     }
   }
 
+  @Test
+  public void testCompare_different_canvas() throws Exception {
```

Comment:
Please rename this test to meet the [convention](https://github.com/Cognifide/aet/blob/master/CONTRIBUTING.md#tests-naming-convention) e.g. to:
`compare_differentHeightScreenshots_expectHeightDifferenceMarkedWithYellow` .